### DISCUSSION TITLE
orchardcore logoff link error !

### DISCUSSION THREAD

**Discussion by MRSheikh** [OrchardCMS/OrchardCore#8164] [Q&A] [answered]
i want to logoff my user with html link and when click on href="/Users/LogOff" error occurred.
```
An error occurred while executing this request.
```

**Comment by hishamco** [comment]
Use a form with a POST

```html
<form asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="LogOff" method="post">
 <button type="submit">@T["Log off"]</button>
</form>
```

**Reply by MRSheikh** [reply]
> 
> 
> Use a form with a POST
> 
> ```
> <form asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="LogOff" method="post">
>  <button type="submit">@T["Log off"]</button>
> </form>
> ```
i used that and a blank white page appears !!
i used that in liquid page.

**Reply by Skrypt** [reply]
I can't repro this issue with the current latest dev branch. If you are doing customizations then provide repro steps.

**Reply by hishamco** [reply]
Please check the network tab in your browser and send the error message

**Reply by MRSheikh** [reply]
> 
> 
> Please check the network tab in your browser and send the error message

![image](https://user-images.githubusercontent.com/17869166/103158184-719fcd80-47d0-11eb-9691-2dbb94329570.png)

**Reply by hishamco** [reply]
Check the 400 error message, the POST should call the `AccountController`, I will have a try

**Reply by MRSheikh** [reply]
> 
> 
> Use a form with a POST
> 
> ```
> <form asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="LogOff" method="post">
>  <button type="submit">@T["Log off"]</button>
> </form>
> ```

i think we should pass anti forgery token with form tag? is that true?

**Reply by hishamco** [reply]
Ya

**Comment by MRSheikh** [comment, marked as answer]
yes!! that's works fine. 
it's about anti forgery token.
``` 
<form class="" method="post" action="/Users/LogOff">
                      <button class="btn" type="submit">خروج</button>
                      {% antiforgerytoken %}
</form>
```
